### PR TITLE
fix(region-info): ap-northeast-3 data not correctly registered

### LINKED ISSUE #13561: (region-info): aws-entities.ts AWS_REGIONS missing ap-northeast-3
[`@aws-cdk/region-info/build-tools/aws-entities.ts`](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/region-info/build-tools/aws-entities.ts#L6-L33) doesn't have `ap-northeast-3` in `AWS_REGIONS` list, while `ap-northeast-3` region has some facts hardcoded in `fact-tables.ts` ([example](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/region-info/build-tools/fact-tables.ts#L101)).

### Reproduction Steps

In my case, I tried to enable load balancer access logging and encountered an error. Example:

```typescript
// loadBalancer is an ApplicationLoadBalancer instance
// logBucket is a Bucket instance
loadBalancer.logAccessLogs(logBucket);
```

### What did you expect to happen?

Expect no error.

### What actually happened?

When building the templates, I got `Error: Cannot enable access logging; don't know ELBv2 account for region ap-northeast-3`

### Environment

These are more or less irrelevant to the issue as I already know what's missing.

  - **CDK CLI Version  :** 1.91.0
  - **Framework Version:** 1.80.0
  - **Node.js Version:** node-v12.18.3
  - **OS               :** MacOS
  - **Language (Version):** TypeScript (3.9.7)

### Other

I'm also wondering if there is a particular reason why `ap-northeast-3` is not included right now and if there is a plan to include that in the future.

--- 

This is :bug: Bug Report

### PR DESCRIPTION
**Pull request by RomainMuller**
The region information for ap-northeast-3 was not correctly registered
as the region was missing from the `AWS_REGIONS` list in the
`aws-entities.ts` file.

This addresses the gap, and adds a validation at the beginning of
`generate-static-data.ts` to ensure no "new" region is introduced here
without also being introduced in the `AWS_REGIONS` list.

Fixes #13561

Credits to @robertd who had a draft PR with similar changes, which I
only saw once it was too late. I've retro-fitted all the good ideas they had
which I did not have on first intention - so thank you @robertd.

Co-Authored-By: @robertd 

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/region-info/build-tools/aws-entities.ts [hunk 1]
```diff
@@ -4,32 +4,33 @@
...
   'sa-east-1',
+  'us-east-1',
+  'us-east-2',
+  'us-gov-east-1',
+  'us-gov-west-1',
+  'us-iso-east-1',
+  'us-isob-east-1',
+  'us-west-1',
+  'us-west-2',
 ].sort();
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/aws-entities.ts:34]: Added the missing region, and also re-ordered entries in alphanumeric order, so it's easier to browse visually (especially scanning for a missing region, etc...).

File: packages/@aws-cdk/region-info/build-tools/fact-tables.ts [hunk 2]
```diff
@@ -46,29 +46,29 @@ export const AWS_CDK_METADATA = new Set([
  * @see https://docs.aws.amazon.com/general/latest/gr/rande.html#s3_website_region_endpoints
  */
 export const ROUTE_53_BUCKET_WEBSITE_ZONE_IDS: { [region: string]: string } = {
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/fact-tables.ts:48]: Ordered map entries alphanumerically to facilitate scanning through it by a human.

File: packages/@aws-cdk/region-info/build-tools/fact-tables.ts [hunk 3]
```diff
@@ -83,61 +83,61 @@ export const PARTITION_MAP: { [region: string]: Region } = {
 
 // https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-access-logs.html#access-logging-bucket-permissions
 export const ELBV2_ACCOUNTS: { [region: string]: string } = {
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/fact-tables.ts:85]: Ordered map entries alphanumerically to facilitate scanning through it by a human.

File: packages/@aws-cdk/region-info/build-tools/fact-tables.ts [hunk 4]
```diff
@@ -83,61 +83,61 @@ export const PARTITION_MAP: { [region: string]: Region } = {
...
   'us-gov-east-1': '190560391635',
-  'cn-north-1': '638102146993',
-  'cn-northwest-1': '037604701340',
+  'us-gov-west-1': '048591011584',
+  'us-west-1': '027434742980',
+  'us-west-2': '797873946194',
 };
 
 // https://aws.amazon.com/releasenotes/available-deep-learning-containers-images
 export const DLC_REPOSITORY_ACCOUNTS: { [region: string]: string } = {
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/fact-tables.ts:114]: Ordered map entries alphanumerically to facilitate scanning through it by a human.

File: packages/@aws-cdk/region-info/build-tools/fact-tables.ts [hunk 5]
```diff
@@ -83,61 +83,61 @@ export const PARTITION_MAP: { [region: string]: Region } = {
...
+  'me-south-1': '217643126080',
+  'sa-east-1': '763104351884',
+  'us-east-1': '763104351884',
+  'us-east-2': '763104351884',
+  'us-west-1': '763104351884',
+  'us-west-2': '763104351884',
 };
 
 // https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
 export const APPMESH_ECR_ACCOUNTS: { [region: string]: string } = {
```
Comment (RomainMuller) [inline packages/@aws-cdk/region-info/build-tools/fact-tables.ts:138]: Ordered map entries alphanumerically to facilitate scanning through it by a human.

**Comment by robertd** [comment]
@RomainMuller I went to bed after converting this PR to a draft thinking I'd be work on it a bit more today 🤣. I wasn't sure if I should use inline `// comments` or stick with more ` /** standard */` ones, but I'm glad you picked what you liked 😃 . Thanks for doing this 👍

**Review by rix0rrr — APPROVED** [review]

**Comment by RomainMuller** [comment]
> @RomainMuller I went to bed after converting this PR to a draft thinking I'd be work on it a bit more today 🤣. I wasn't sure if I should use inline `// comments` or stick with more ` /** standard */` ones, but I'm glad you picked what you liked 😃 . Thanks for doing this 👍

I prefer on-the-same-line here, for some reason I find it more readable - but definitely a personal thing here... But this was actually a great bonus on your version of the PR, so... 😅

**Pull request merged**